src/components/layout/MainNavigation.js

```diff
@@ -3,7 +3,7 @@ import classes from "./MainNavigation.module.css";
 
 const MainNavigation = () => {
```

Comment:
I think this component should choose a default value after a season is chosen without the user clicking on it for the first time